MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include *.md
```

Comment:
> Are rst files included automatically

Nope. Added in https://github.com/yola/yolapy/commit/76e232ebbbe79c07ca86f41dd0858f9f03f4cabf